src/components/SummaryCard/_mixins.scss

```diff
@@ -0,0 +1,160 @@
+////
+/// Summary card mixins.
+/// @group summary-card
+/// @copyright IBM Security 2019
+////
+
+@import '../../globals/index';
+
+@mixin summary-card {
+  /// Summary card header height.
+  /// @type Length
+  /// @access private
+  $header--height: carbon--mini-units(4);
```

Comment:
For consistency, can we either use keyword arguments (https://sass-lang.com/documentation/at-rules/mixin#keyword-arguments) here or remove parts where we are using them? For example - https://github.com/carbon-design-system/ibm-security/pull/136/files#diff-c20efeb2ceca6394e0d14940502bcfd5R32

Reply:
What do you mean about about consistency etc? 🤔

Reply:
Consistency in how we call methods, for example `carbon--mini-units($count: 4)` versus `carbon--mini-units(4)`